docs/content/guide/migration.ngdoc

```diff
@@ -30,6 +31,967 @@ which drives many of these changes.
+
+Plain Object:
+
+```html
+<!-- Before: -->
+<span ng-bind="myPlainObject">[object Object]</span>
+
+<!-- After: -->
+<span ng-bind="myPlainObject">{'a':1,'b':2}</span>
+```
+
+Object with custom `toString()`:
+
+```html
+<!-- Before: -->
+<span ng-bind="myCustomObject">[object Object]</span>
+
+<!-- After: -->
+<span ng-bind="myCustomObject">a+b</span>
+```
+
+If you want the output of `toString()`, you can call it manually on the value in `ngBind`:
+
+```html
+<span ng-bind="myObject.toString()">[object Object]</span>
+```
+
+
+#### **ngModel**:
+
+<minor />
+Due to [9e24e7](https://github.com/angular/angular.js/commit/9e24e774a558143b3478536911a3a4c1714564ba),
+`NgModelController` now defines its methods on its prototype, instead of on each instance. As a
+consequence, `NgModelController` methods always need to be called in the correct context. For example
+`$scope.$watch('something', myNgModelCtrl.$setDirty)` will no longer work, because the `$setDirty`
+method is passed without any context. The code must now be changed to:
+
+```js
+$scope.$watch('something', function() {
+  myNgModelCtrl.$setDirty();
+})
+```
+
+<hr />
+<minor />
+Due to [7bc71a](https://github.com/angular/angular.js/commit/7bc71adc63bb6bb609b44dd2d3ea8fb0cd3f300b),
+the values returned by synchronous validators are always treated as boolean. Previously, only a
+literal `false` return value would cause the validation to fail. Now, all falsy values will cause
+the validation to fail, as one would naturally expect.
+
+Specifically, the values `0`, `null`, `NaN` and `''` (the empty string) used to cause the validation
+to pass and they will now cause it to fail. The value `undefined` was treated similarly to a pending
+asynchronous validator, causing the validation to be pending. `undefined` is now also treated as
+`false`.
+
+If your synchronous validators are always returning boolean values (which should already be the case
+for most applications anyway), then this change does not affect you. If not, make sure you always
+return a boolean value (`true/false`) indicating whether the input is valid or not.
+
+
+#### **ngModelOptions**:
+
+<major />
+Due to [87a2ff](https://github.com/angular/angular.js/commit/87a2ff76af5d0a9268d8eb84db5755077d27c84c),
+if a setting is not applied on `ngModelOptions`, it will be inherited from the nearest
+`ngModelOptions` ancestor. Previously, it would default to `undefined`.
+
+It is possible that an `ngModelOptions` directive that does not set a property, has an ancestor
+`ngModelOptions` that does set this property to a value other than `undefined`. This would cause the
+`ngModel` and input controls below this `ngModelOptions` directive to display different behaviour.
+If this is not desired, you can explictly set the property in the `ngModelOptions` to prevent it
+from inheriting from the ancestor. For example if you had the following HTML:
+
+```html
+<form ng-model-options="{updateOn: 'blur'}">
+  <input ng-model="..." />
+</form>
+```
```

Comment:
I just realized this example is wrong :( `ngModel` inherits its closest `ngModelOptions` if it doesn't have its own :(